docs/migrate-with-crucible.md

```diff
@@ -27,8 +27,12 @@ For this setup, you'll need:
 ### Seed crucible downstairs with the OS image
 
 From the machine where you will run the crucible downstairs, set up 3 crucible
-downstairs regions using the `--import-path` flag to specify where the OS image
-is. (See the crucible documentation for information about other flags.)
+downstairs regions. Use  the `--import-path` flag to specify where the OS image
+is on the filesystem. Specify the address and port the downstairs will listen
+on using the `-p` and the `-a` flags, respectively. The address may be
```

Comment:
`-a` for address, and `-p` for port, I think you got the order switched here.

Reply:
whoops, thanks